### DISCUSSION TITLE
[Solution found] cannot install Emgu.CV.runtime.windows.dldt via nuget package

### DISCUSSION THREAD

**Discussion by josefbauerMV** [emgucv/emgucv#448] [General]
I am a commercial emgu cv user for years and currently try to update my projects and want to use the nuget packages.

When I try to install the runtime package, I get  this error:

Could not install package 'Emgu.runtime.windows.msvc.rt.x64 19.28.29336'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.7.2', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.				

Surprisingly, I was able to successfully add it to the first project of a solution.

What could be the problem?

Thanks a lot for your help.

EDIT: I found the solution  here:
https://github.com/emgucv/emgucv/issues/399

simply migrate from package.config to package reference by right clicking on package.config in each project and migrate from package.config to PackageReference

**Comment by emgucv** [comment]
Glad to know.
